Library/Homebrew/cli/parser.rb

```diff
@@ -259,7 +259,7 @@ def common_switch(name)
       end
 
       def option_passed?(name)
-        @args.respond_to?(name) || @args.respond_to?("#{name}?")
+        @args[name.to_sym] || @args["#{name}?".to_sym]
```

Comment:
Why this change?

Reply:
`build_from_source` is an instance method now and `respond_to?` would always return `true`.